### PR TITLE
Document redirects in the Handbook

### LINKED ISSUE #2195: Document how to do content redirects
@robdodson should this go in the handbook or elsewhere?

### PR DESCRIPTION
**Pull request by kaycebasques**
Fixes #2195

Changes:

* Adds instructions on how to do redirects.
* Adds a new `Content infrastructure` section to the Handbook.

### CODE REVIEW

File: src/site/content/en/handbook/handbook.11tydata.js [hunk 1]
```diff
@@ -50,6 +49,13 @@ module.exports = {
           'handbook/self-assessment-components',
         ],
       },
+      {
+        title: 'Content infrastructure',
+        pathItems: [
+          'handbook/resolving-glitch-issues',
+          'handbook/redirects',
+        ],
+      },
```
Comment (kaycebasques) [inline src/site/content/en/handbook/handbook.11tydata.js:58]: @samthor or @robdodson need a review of this JS change

Comment (samthor) [reply src/site/content/en/handbook/handbook.11tydata.js:58]: Actually, I know I approved, but could you make this change? It basically wants you to put them on one line:

```js
        pathItems: ['handbook/resolving-glitch-issues', 'handbook/redirects],
```

Comment (kaycebasques) [reply src/site/content/en/handbook/handbook.11tydata.js:55]: Yup yup already doing it

**Comment by kaycebasques** [comment]
@samthor Percy is saying that the Contributor page is affected by this change, but I'm not seeing anything...

**Comment by robdodson** [comment]
> @samthor Percy is saying that the Contributor page is affected by this change, but I'm not seeing anything...

it might be the case that it didn't snapshot that page properly when it landed on master. Let's approve it and see what happens :D

**Pull request merged**